massgov/mayflower-patternlab
# DP-8378 banner display fix in mobile

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description


## Related Issue / Ticket

- [DP-8378 The page banner display incorrectly for topic page within mobile responsive](https://jira.state.ma.us/browse/DP-8378)

## Steps to Test


1. Open pages contain `div.ma__page-banner__container`,  Topic, Service and Org pages.
2. Check the header banner display in different window sizes.
- Banner background shapes 
- Not only the page title, but make sure the banner description is fully displayed.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
